src/commcare_cloud/ansible/roles/elasticsearch/templates/monit.elasticsearch.j2

```diff
@@ -0,0 +1,6 @@
+check file elasticsearch with path {{ elasticsearch_data_dir }}/logs/icds-2.0.log
+   group database
+   group elasticsearch
+   start program = "/bin/systemctl start {{ elasticsearch_service_name }}"
+   stop program = "/bin/systemctl stop {{ elasticsearch_service_name }}"
+   if content = "java.lang.OutOfMemoryError" then restart
```

Comment:
Will this not trigger again and again, since the logfile will continue to have this content in it? It should search the status, not the log file, right?

Reply:
it will not trigger multiple times for the same occurrence in logfile
ref: https://mmonit.com/monit/documentation/monit.html#FILE-CONTENT-TEST